assets/lift_inspector/acquia_lift_inspector.js

```diff
@@ -0,0 +1,125 @@
+          '             <div><label>' + Drupal.t('Rule Name') + '</label><p class="liftValue"><%- decision.rule_name %></p></div>' +
+          '             <div><label>' + Drupal.t('Content Displayed') + '</label><p class="liftValue"><%- decision.content_name %></p></div>' +
+          '           </div>' +
+          '          </div>' +
+          '        <% }); %>' +
+          '      <% } else { %>' +
+          '        <div class="item-wrapper"><p>' + Drupal.t('No decisions made.') + '</p></div>' +
+          '      <% } %>' +
+          '    </div>' +
+          '  </div>' +
+          '  <div class="lift-inspector-wrapper">' +
+          '    <h4>' + Drupal.t('Recent captures') + '</h4>' +
+          '    <div class="captures container">' +
+          '      <% if (!_.isEmpty(lastDecisionRequest.captures)) { %>' +
+          '        <% _.each(lastDecisionRequest.captures, function (capture) {%>' +
+          '          <div class="lastdecision-item-wrapper item-wrapper">' +
+          '          <div class="item-title"><h3><%- capture.event_name %></h3></div>' +
+          '           <div class="item-details">' +
+          '          <div><label>' + Drupal.t('Source') + '</label><p class="liftValue"><%- capture.event_source %></p></div>' +
+          '          <div class="inline-wrap"><label class="inline-pre">' + Drupal.t('URL') + '</label><pre class="liftValue"><%- capture.url %></pre></div>' +
+          '          <div> <label>' + Drupal.t('Date') + '</label><p class="liftValue"><%- capture.event_date %></p></div>' +
+          '           </div>' +
+          '          </div>' +
+          '        <% }); %>' +
+          '      <% } else { %>' +
+          '        <div class="item-details"><p class="liftValue">' + Drupal.t('No recent captures.') + '</p></div>' +
+          '      <% } %>' +
+          '    </div>' +
+          '  </div>' +
+          '</div>'
+        );
+
+        if (typeof window.AcquiaLift === 'undefined') {
+          $modal_element = '<h1 id="lift-inspector">' + Drupal.t('Lift is not currently loaded.') + '</h1>';
+        } else {
+          $modal_element = $(modal_template(window.AcquiaLift));
+        }
+
+        var dialogActiveClassName = 'dialog-active';
+        var dialogContainerSelector = 'body';
+        Drupal.dialog($modal_element, {
+          title: Drupal.t('Lift Inspector'),
+          show: { effect: 'fadeIn', duration: 200 },
+          hide: { effect: 'fadeOut', duration: 200 },
+          dialogClass: 'lift-inspector-dialog',
+          minWidth: 900,
+          create: function(event, ui) {$(dialogContainerSelector).addClass(dialogActiveClassName);},
+          beforeClose: function(event, ui) {$(dialogContainerSelector).removeClass(dialogActiveClassName);}
+        }).showModal();
+      }
+
+      // $(document).ready(function(){
+      //   show_inspector_modal();
+      // })
+
+
+      $(document).keypress(function (e) {
```

Comment:
Yes, I tested it in most browsers besides IE, but according to the docs, its supported.